WordPress/gutenberg
# Update skip serialization docs to reflect ability to target specific properties

## What?


Updates the docs on Block Supports to reflect the ability to target individual Block Supports properties via the array syntax on `__experimentalSkipSerialization`.

Support for this was added in https://github.com/WordPress/gutenberg/pull/36293 (and shipped in `13.0.0`) thanks to great work from @aaronrobertshaw.

## Why?


The docs were out of date.


## How?


- Removes reference inability to target individual properties.
- Adds example of targeting specific properties only.

## Testing Instructions





- Read docs for comprehension and spelling/grammar. 
- Check I got things right by referencing the PR which added support.
- Check the example is correct.
- Profit!

## Screenshots or screencast
